Config/CMakeLists.examples.txt

```diff
@@ -33,12 +33,12 @@ add_executable(example_dream         example_dream_01.cpp
 target_link_libraries(example_sparse_grids  Tasmanian::Tasmanian)
 target_link_libraries(example_dream         Tasmanian::Tasmanian)
 
-# if Fortran is available, then add the Fortran example
-if (TARGET Tasmanian_libfortran90)
+# if target Tasmanian::Fortran90 is present,
+# then Tasmanian was build with Fortran support
+if (TARGET Tasmanian::Fortran90)
     enable_language(Fortran)
     add_executable(example_sparse_grids_f90  example_sparse_grids.f90)
-    target_link_libraries(example_sparse_grids_f90  Tasmanian_libfortran90)
+    # Tasmanian::Tasmanian and Tasmanian::Fortran90 are equivalent even for CXX code
+    target_link_libraries(example_sparse_grids_f90  Tasmanian::Tasmanian)
```

Comment:
@dalg24 Question:

Now I create an imported target `Tasmanian::Tasmanian` but what is a good target name so the user can specify `shared` or `static` (in case both libs are present are present).
* `Tasmanian::Tasmanian_shared`
* `Tasmanian::TasmanianShared`
* `Tasmanian::shared` or `Tasmanian::Shared`
* `Tasmanian::Tasmanian::shared`

Reply:
I'll go with `Tasmanian::shared` and `Tasmanian::static`.